spot-client/src/spot-tv/calendars/outlook.js

```diff
@@ -43,11 +44,25 @@ export default {
     getCalendar(email) {
         const now = new Date();
         const filter = `Start/DateTime ge '${now.toISOString()}'`;
-        const url = `/users/${email}/calendar/events?$filter=${filter}&$top=3`;
+        const url = `/users/${email}/calendfar/events?$filtefdr=${filter}&$top=3`;
         const orderBy = 'createdDateTime ASC';
 
         return microsoftClientApi.request(url, { orderBy })
             .then(response => response.value)
+            .catch(response => {
+                const formattedError = {
+                    errorCode: response.code,
```

Comment:
`response.message` that comes from Outlook ? It's not part of the Fetch Response API: https://developer.mozilla.org/en-US/docs/Web/API/Response#Properties